### ISSUE TITLE
curl_xml: Tables broken

### ISSUE THREAD

**Issue by faxm0dem** [collectd/collectd#931]
Commit 0afea60 broke the following config:
```xml
<Plugin "curl_xml">
  <URL "file:///tmp/foo.xml">
    Host "foo"
    Instance "GridEngine"
    <xpath "/GridEngineBasicCount/Jobs">
      Type "gauge"
      InstanceFrom "@Instance"
      ValuesFrom "Count/text()"
    </xpath>
  </URL>
</Plugin>
```
foo.xml:
```xml
<?xml version="1.0"?>
<GridEngineBasicCount MasterName="foo" Date="02/08/2015-21:59:01">
  <Jobs Instance="Pending">
    <Count>13021</Count>
  </Jobs>
  <Jobs Instance="Running">
    <Count>19037</Count>
  </Jobs>
</GridEngineBasicCount>
```
Error message:
```
curl_xml plugin: Base-XPath /GridEngineBasicCount/Jobs is a table (more than one result was returned), but no instance-XPath has been defined.
```

Please revert

**Comment by trtrmitya** [comment]
Any progress so far?  curl_xml is indeed broken now.

**Comment by mfournier** [comment]
A fix for this issue will be part of 5.3.3, 5.4.3 and 5.5.0. Closing.
